native/CMakeLists.txt

```diff
@@ -1518,6 +1639,19 @@ cocos_source_files(
     NO_WERROR    cocos/bindings/auto/jsb_scene_auto.cpp
                  cocos/bindings/auto/jsb_scene_auto.h
 )
+endif()
+if(OPENHARMONY)
```

Comment:
`else()`

Reply:
It's temporary at the moment and will open later